File: Easy/Java/1614. Maximum Nesting Depth of the Parentheses.java
class Solution {
    public int maxDepth(String s) {
        int depth = 0;
        int open = 0;

        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == '('){
                open++;
                depth = Math.max(depth, open);
            }
            if(s.charAt(i) == ')')
                open--;
        }

        return depth;
    }
}

File: Easy/Python/1614. Maximum Nesting Depth of the Parentheses.py
class Solution(object):
    def maxDepth(self, s):
        depth = 0
        open_paren = 0

        for i in range(len(s)):
            if s[i] == '(':
                open_paren += 1
                depth = max(depth, open_paren)
            if s[i] = ')':
                open_paren -= 1
        
        return depth
